Index: src/Pclass1.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Pclass1.java	(date 1502877319000)
+++ src/Pclass1.java	(revision )
@@ -2,5 +2,9 @@
 
     public static void main(String[] args) {
         System.out.println("test");
+        System.out.println("test1");
+        System
     }
 }
+
+
